index.html

```diff
@@ -610,57 +598,79 @@
-            <code>property="schema:author"</code> or <code>property="schema:contributor"</code>,
-            <code>typeof="schema:Person"</code>, and a <code>resource</code> attibute that is a URL
-            identifying the person (typically their home page, or possibly email address — it could
-            also be a URL to their ORCID).
+            <code>property="schema:author"</code> or <code>property="schema:contributor"</code> and a
+            <code>typeof="sa:ContributorRole"</code>.
           </p>
           <p>
-            Inside of that <code>li</code> arbitrary properties of <code>schema:Person</code> can be
-            specified, but providing at least <code>schema:givenName</code> and
-            <code>schema:familyName</code>, as well as ideally <code>schema:email</code> is very
-            much recommended.
+            A <code>sa:ContributorRole</code> type (following the
+            semantic of
+            schema.org <a href="http://schema.org/Role">Role</a>) is
+            used so that affiliations or contact informations (email
+            address, etc.) relevant to this specific scholarly article
+            (and this specific scholarly article only) can be
+            specified. This is important as authors may have different
+            affiliations and contact points at the time they are
+            publishing a scholarly article but may want to specify
+            only a subset of those. Readers not familiar with the
+            semantic of schema.org Role can consult
+            the <a href="http://blog.schema.org/2014/06/introducing-role.html">introductory
+            blog post</a>.
           </p>
           <p>
-            If there is an affiliations <code>ol</code> and a given author is affiliated, there must
-            be an <code>a</code> element with its <code>href</code> pointing to that affiliation, a
-            <code>resource</code> matching the URL identifying the affiliation, and
-            <code>property="schema:affiliation"</code>. The content of that <code>a</code> element
-            must be a string that matches the one that will be generated by CSS to label the
-            affiliation; Latin lowercase being recommended. (This is a hack, but we can only do so
-            much within the limits of CSS — better <code>counter</code> would be needed.)
+            Inside of that <code>li</code>, arbitrary properties
+            of <code>schema:Person</code>, filling
+            the <code>schema:author</code> property of
+            the <code>sa:ContributorRole</code> can be specified, but
+            providing at least <code>schema:givenName</code> and
+            <code>schema:familyName</code>. It is recommended that
+            these properties are wrapped into an hyperlink
+            identifying the person with a URL to their home page,
+            their <a href="http://orcid.org/">ORCID</a>, or an email
+            address.
```

Comment:
Don't know if you saw but I added some comments to the PR. Happy to detail more if the comments are unclear. In short, I don't think (but I am really not sure) that it's a good idea to expose the email in HTML when a contributor is not a corresponding author (there can be many corresponding authors). For what I understand if a contributor is willing to be contacted about the article, he is one of the corresponding authors (and so in the current version of scholarly HTML he will have a little email icon containing the `schema:email` prop). IMHO if a contributor is not willing to be contacted, we should not expose his email in HTML (but we can expose general metadata about the Person (not part of the contributorRole) in meta tags in RDFa or JSON-LD). In the PR comments I added some examples taken from eLife that seem to follow this approach but I haven't investigated in great details.

For ORCID _and_ homepage, happy to add examples, can't see any issue with that.